.github/workflows/patch-mariadb-tests.yml

```diff
@@ -102,4 +102,13 @@ jobs:
           cd ~/frappe-bench/
           wget https://frappeframework.com/files/v10-frappe.sql.gz
           bench --site test_site --force restore ~/frappe-bench/v10-frappe.sql.gz
+          cd apps/frappe && git remote set-url upstream https://github.com/frappe/frappe.git && git fetch --all --tags
+          git tag --sort version:refname \
+            | grep -v "beta" \
+            | tail -1 \
+            | xargs -L 1 git checkout -f
```

Comment:
This might check out to the latest v12 or v13 releases randomly?

Reply:
Nope, it shouldn't. `version` sort in git considers "semantic" sorting and not date based, so the last released tag will be picked up (ignoring the ones with `beta` in tag name) 

You can run it locally if you have all the tags.  `git tag --sort version:refname`


ref: https://git-scm.com/docs/git-tag#Documentation/git-tag.txt---sortltkeygt